packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -83,7 +83,7 @@ const InvalidDashboardChartTile: FC = () => (
 
 type Props = Pick<
     React.ComponentProps<typeof TileBase>,
-    'tile' | 'onEdit' | 'onDelete' | 'isEditMode'
+    'tile' | 'onEdit' | 'onDelete' | 'isEditMode' | 'onDuplicate'
```

Comment:
what does this do ?

Reply:
This adds onDuplicate to the props of the tiles.